scipy/stats/tests/test_fit.py

```diff
@@ -101,13 +101,28 @@ def test_cont_fit(distname, arg, method):
         with np.errstate(all='ignore'):
             rvs = distfn.rvs(size=fit_size, *arg)
             est = distfn.fit(rvs, method=method)  # start with default values
-            if method.lower() == 'mle':
-                data = stats.CensoredData(rvs, rvs)
-                est2 = distfn.fit(data)
-                if not np.allclose(est2, est, atol=1e-9):
-                    raise AssertionError(
-                        'Different results when fitting trivially censored'
-                        f' data: {distfn.name}: est={est} est2={est2}')
+            if (method.lower() == 'mle' and
+                    distfn.name not in ['chi', 'genhalflogistic', 'rdist']):
```

Comment:
Would today's changes to `chi` make it pass?

Reply:
Unfortunately no.  The `chi` PR came out of looking into this, but it turned out it didn't help.

Reply:
To be clear: the "failure" here doesn't mean the calculation is incorrect.  It just means that for the given number of interval-censored points and the given interval sizes, the relative difference in the result for the censored and uncensored data is more than 0.05.  If I plot the cross-section of the NLF for the `chi` test with the censored data, I can see that the result is clearly at a minimum.  Also, if the factors that set the interval sizes are changed from [0.975, 1.025] to [0.99, 1.01], the test passes.